drivers/gpu/drm/tests/drm_framebuffer_test.c

```diff
@@ -7,6 +7,8 @@
 
 #include <kunit/test.h>
 
+#include <vdso/bits.h>
+
```

Comment:
Is this needed?

Reply:
It's implicitly included, but I was in doubt of if it's ok to use macros (in this case, the BIT macro) in that way, so I included its definition. Should I remove it?

Reply:
I believe you could just remove it.

Reply:
> I believe you could just remove it.

Did that in the last forced-push, thanks!